### DISCUSSION TITLE
problem encountered with torch_tensorrt

### DISCUSSION THREAD

**Discussion by arielsolomon** [pytorch/TensorRT#1146] [Q&A]
Hi,
I am trying to use torch_tensorrt in docker container (nvcr.io/nvidia/tensorrt:22.05-py3 and also in nvcr.io/nvidia/tensorrt:22.04-py3) with error repeating regardless the docker image I used or even different gpu models (rtx3090, rtx2060) when trying to import torch_tensorrt:
>>> import torch_tensorrt
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.8/dist-packages/torch_tensorrt/__init__.py", line 11, in <module>
    from torch_tensorrt._compile import *
  File "/usr/local/lib/python3.8/dist-packages/torch_tensorrt/_compile.py", line 2, in <module>
    from torch_tensorrt import _enums
  File "/usr/local/lib/python3.8/dist-packages/torch_tensorrt/_enums.py", line 1, in <module>
    from torch_tensorrt._C import dtype, DeviceType, EngineCapability, TensorFormat
ImportError: libtorch_cuda_cu.so: cannot open shared object file: No such file or directory
We tried different ways to install torch_tensorrt and its dependencies with no luck.
Would appreciate some help.

**Comment by narendasan** [comment]
Can you provide the exact versions of the libraries you have installed in the container and how you installed them? Specifically PyTorch, Torch-TensorRT and TensorRT?

Also we have a dockerfile based on ngc/tensorrt available in `//docker` which you could use as well

**Reply by arielsolomon** [reply]
Thanks for your reply.
Problem solved after uninstalling and reinstalling the compatible torch packages to cuda driver
